src/instance.js

```diff
@@ -764,9 +764,16 @@ Instance.prototype.getTablesStream = common.paginator.streamify('getTables');
  * Set the instance metadata.
  *
  * @param {object} metadata Metadata object.
- * @param {string} metadata.displayName The descriptive name for this
- *     instance as it appears in UIs. It can be changed at any time, but
- *     should be kept globally unique to avoid confusion.
+ * @param {Object} metadata.instance
+ * The Instance which will (partially) replace the current value.
+ *
+ * This object should have the same structure as [Instance]{@link google.bigtable.admin.v2.Instance}
+ * @param {Object} metadata.updateMask
```

Comment:
`updateMask` should be optional.  The default ought to be to update all fields.

Reply:
Updated as suggested. Please review.